### PR TITLE
Make News & Blogs section dynamic

### PR DESCRIPTION
**Pull request by kayceeDev**
Fixes: #128

### CODE REVIEW

**Comment by FeynmanZhou** [comment]
@kayceeDev Thanks for contributing to the Notary website and fixing #128 . Would you mind following this [Figma](https://www.figma.com/file/sV3L1CedpMhuO6qXyaipbV/Notary-website-landing-page?type=design&node-id=0-1&t=qltOafVo1GsWrcP8-0) to improve the **Blogs & News** section's layout?  I would like to display four latest blogs on the landing page like the following screenshot. I am fine we can improve it in a follow-up PR and merge this one first.

image

**Comment by kayceeDev** [comment]
> @kayceeDev Thanks for contributing to the Notary website and fixing #128 . Would you mind following this [Figma](https://www.figma.com/file/sV3L1CedpMhuO6qXyaipbV/Notary-website-landing-page?type=design&node-id=0-1&t=qltOafVo1GsWrcP8-0) to improve the **Blogs & News** section's layout?  I would like to display four latest blogs on the landing page like the following screenshot. I am fine we can improve it in a follow-up PR and merge this one first.
> 
> image
> 

Thank you for your review @FeynmanZhou I will improve this PR by adding the right images for each blog since these assets are not present in the repository. To make this more flexible we can also have a placeholder image for blogs without images. We can improve the docs further in a different PR to state the standard for future blog frontmatter to include image url.

**Comment by FeynmanZhou** [comment]
@kayceeDev Sounds good to me! Let us know if you improved and finished it in this PR. Thanks!

**Comment by kayceeDev** [comment]
> @kayceeDev Sounds good to me! Let us know if you improved and finished it in this PR. Thanks!

@FeynmanZhou Take a look at the latest implementation and tell me what you think. Looking forward to your feedback. Thanks.

**Comment by kayceeDev** [comment]
@FeynmanZhou @thisisobate I was considering making each blog box clickable. This favors UX.  I am open to your suggestions. Thanks

**Comment by thisisobate** [comment]
> I was considering making each blog box clickable

Sounds like a god idea 👍

File: content/en/_index.html [hunk 1]
```diff
@@ -216,9 +186,7 @@ <h1 class="text-left">
       <div class="feature-description text-left">
         <div class="description is-bullets">
           <ul>
-            <li>
-              100% open source, built and improved by the active communitye
-            </li>
+            <li>100% open source, built and improved by the active communitye</li>
```
Comment (FeynmanZhou) [inline content/en/_index.html:189]: Could you help to fix typo? communitye=> community

**Comment by FeynmanZhou** [comment]
> @FeynmanZhou @thisisobate I was considering making each blog box clickable. This favors UX. I am open to your suggestions. Thanks

Sounds good. Do you think we can improve it in a follow-up PR?

**Comment by FeynmanZhou** [comment]
cc @yizha1 for review

**Comment by kayceeDev** [comment]
> > @FeynmanZhou @thisisobate I was considering making each blog box clickable. This favors UX. I am open to your suggestions. Thanks
> 
> Sounds good. Do you think we can improve it in a follow-up PR?

Yes definitely.

I have included blog images on this PR and dynamically added `image-alt` to each blog frontmatter.

This should be good now.

**Review by FeynmanZhou — APPROVED** [review]
LGTM

File: content/en/blog/2023/announcing-notation-rc4.md [hunk 2]
```diff
@@ -3,6 +3,8 @@ title: Notation v1.0.0-RC.4 is available!
 author:  "Notation Release Team"
 date: 2023-04-21
 draft: false
+image: "rc-latest.svg"
```
Comment (yizha1) [inline content/en/blog/2023/announcing-notation-rc4.md:6]: This svg is not good, can we use the default one?

File: content/en/blog/2023/announcing-notation-rc3.md [hunk 3]
```diff
@@ -3,6 +3,8 @@ title: Notation v1.0.0-RC.3 is available!
 author:  "Notation Release Team"
 date: 2023-03-08
 draft: false
+image: "rc-previous.svg"
```
Comment (yizha1) [inline content/en/blog/2023/announcing-notation-rc3.md:6]: This svg is not good, can we use the default one?

**Review by yizha1 — COMMENTED** [review]
@kayceeDev Thanks. But the svg doesn't look good. I suggest keeping current one.

**Comment by FeynmanZhou** [comment]
@kayceeDev  The layout in my 13'inch laptop looks not comfortable. Is it possible to adjust its layout and size?
image

File: content/en/blog/2023/announcing-notation-rc4.md (hunk 2 above)
Comment (kayceeDev) [reply content/en/blog/2023/announcing-notation-rc4.md:6]: > @kayceeDev The layout in my 13'inch laptop looks not comfortable. Is it possible to adjust its layout and size? image

Are you referring to the vertical card or the horizontal one?

**Comment by kayceeDev** [comment]
> @kayceeDev The layout in my 13'inch laptop looks not comfortable. Is it possible to adjust its layout and size? image

sure

**Review by FeynmanZhou — APPROVED** [review]
LGTM

**Review by FeynmanZhou — APPROVED** [review]
LGTM

**Comment by FeynmanZhou** [comment]
@yizha1 Could you re-approve this PR? Thanks

**Review by yizha1 — APPROVED** [review]
LGTM. Thanks @kayceeDev

**Comment by yizha1** [comment]
@kayceeDev you need to sign the commits, see https://docs.github.com/en/authentication/managing-commit-signature-verification/about-commit-signature-verification

**Comment by kayceeDev** [comment]
All my commits where signed..

**Comment by Sumitmaithani** [comment]
> All my commits were signed..

you can refer to [commit signatures verification](https://docs.github.com/en/authentication/managing-commit-signature-verification/about-commit-signature-verification) to sign all your commits

**Comment by yizha1** [comment]
@kayceeDev It seems not, you can scroll to the bottom of this PR, see 

image

**Pull request merged**